tesla-hero-slider/src/components/slide-params.js

```diff
@@ -0,0 +1,70 @@
+import React, {Component} from 'react';
+import propTypes from 'prop-types';
+import AnimateValue from './animate-value';
+import './../styles/scss/components/slide-params.css';
+
+let DELAY_TOP_SPEED = 200,
+	DELAY_MPH = 700,
+	DELAY_MILE_RANG = 1200;
+
+class SlideParams extends Component {
+
+	static propTypes = {
+		activeCar: propTypes.object.isRequired,
+		animationForward: propTypes.bool.isRequired
+	};
+
+	componentWillReceiveProps(props) {
+		if (!props.animationForward) {
+			DELAY_TOP_SPEED = 1200;
+			DELAY_MILE_RANG = 200;
+		}
+	}
+
+	render() {
+		const {activeCar, animationForward} = this.props;
+
+		return (
+			<div className='tesla-slide-params'>
+				<ul className='tesla-slide-params__list'>
+					<li className='tesla-slide-params__item'>
+						<AnimateValue
+							className='tesla-slide-params__value'
+							animationForward={animationForward}
+							value={activeCar.topSpeed}
+							delay={DELAY_TOP_SPEED}
+							prefix="+"
```

Comment:
Don't add prefix and suffix to AnimateValue... Just render it before and after this component